packages/core/src/components/SortCarets/index.tsx

```diff
@@ -7,88 +7,114 @@ export type Props = {
   /** Whether or not to display the bottom caret. */
   down?: boolean;
   /** If enabled, the caret is more pronounced. */
-  enableDown: boolean;
+  enableDown?: boolean;
   /** If enabled, the caret is more pronounced. */
-  enableUp: boolean;
+  enableUp?: boolean;
   /** Whether or not to display the top caret. */
   up?: boolean;
 };
 
-/** Carets to indicate sorting on DataTable. */
+/** Carets to indicate sorting on a table. */
 export class SortCarets extends React.Component<Props & WithStylesProps> {
   static defaultProps = {
-    down: true,
+    down: false,
     enableDown: false,
     enableUp: false,
-    up: true,
+    up: false,
   };
 
   renderCaretUp() {
-    const { cx, down, up, enableUp, styles } = this.props;
+    const { cx, up, enableUp, styles } = this.props;
 
     return (
       up && (
         <span
           className={cx(
-            down && styles.caret_up,
             styles.caret,
+            styles.caret_up,
             enableUp ? styles.caret_active : styles.caret_inactive,
           )}
         >
-          <IconCaretUp decorative size="2em" />
+          <IconCaretUp decorative size="1.6em" />
         </span>
       )
     );
   }
 
   renderCaretDown() {
-    const { cx, down, up, enableDown, styles } = this.props;
+    const { cx, down, enableDown, styles } = this.props;
 
     return (
       down && (
         <span
           className={cx(
-            up && styles.caret_down,
             styles.caret,
+            styles.caret_down,
             enableDown ? styles.caret_active : styles.caret_inactive,
           )}
         >
-          <IconCaretDown decorative size="2em" />
+          <IconCaretDown decorative size="1.6em" />
         </span>
       )
     );
   }
 
   render() {
-    const { cx, styles } = this.props;
+    const { cx, styles, up, down } = this.props;
 
     return (
-      <span className={cx(styles.caret_container)}>
+      <span className={cx(styles.container, up && down && styles.container_full)}>
         {this.renderCaretUp()}
         {this.renderCaretDown()}
       </span>
     );
   }
 }
 
-export default withStyles((theme: WithStylesProps['theme']) => ({
-  caret_container: {
-    marginRight: 0.5 * theme!.unit,
+export default withStyles(({ color, unit }) => ({
+  container: {
+    display: 'inline-block',
+    verticalAlign: 'middle',
+    marginTop: -1,
```

Comment:
why `-1` here?

Reply:
For the most part, our regular text is 15px while the sort carets are 16px, so it's mainly to just offset that and align it slightly better.